models/articles_models.js

```diff
@@ -1,7 +1,24 @@
 const db = require("../db/connection");
 
-exports.selectArticles = (topic) => {
+exports.selectArticles = (sort_by = "created_at", order = "desc", topic) => {
+  const validSortByValues = [
+    "article_id",
+    "title",
+    "topic",
+    "author",
+    "body",
+    "created_at",
+    "votes",
+    "comment_count",
+  ];
+  const validOrderValues = ["asc", "desc"];
   const validTopicValues = ["mitch", "cats", undefined];

```

Comment:
for `validTopicValues` check the devData, you will find there are more topics in the devData to be included - the dev data is what will actually be used when you come to apply this project to Front End